README.md

```diff
@@ -1,2 +1,91 @@
 # DocToolsBundle
 Helper tools for documentation
+
+## Installation
+
+As usual, there are a few steps required to install this bundle:
+
+1.a **Add this bundle to your project using composer**:
+
+```bash
+$ composer require prestashop/doc-tools-bundle
+```
+
+1.b **Add this bundle to your project as a composer dependency (from your forked repository)**:
+
+```javascript
+    // composer.json
+    {
+        // ...
+        require-dev: {
+            // ...
+            "prestashop/doc-tools-bundle": "dev-my-branch"
+        },
+        // ...
+        "repositories": [
+            // ...
+            {
+                "type": "vcs",
+                "url": "https://github.com/myfork/DocToolsBundle",
```

Comment:
We could distribute this project as packagist or even try github package?

Reply:
It's not required to install the bundle, actually this extra line is only here when you need to override the repo with your fork (which you have to do when you are developing on a branch)

Reply:
Precision: it's not needed as long as you want to use the `main` branch Composer automatically find the repository